### ISSUE TITLE
module 'syft' has no attribute 'load'

### ISSUE THREAD

**Issue by rkarn** [OpenMined/PySyft#5668]
I run the  notebook Tutorial_0_TenSEAL_Syft_Data_Owner   . I get following error.  

AttributeError                            Traceback (most recent call last)
<ipython-input-1-a81a3ddffd44> in <module>
      3 import pytest
      4 
----> 5 sy.load("tenseal")

AttributeError: module 'syft' has no attribute 'load'

![image](https://user-images.githubusercontent.com/20768904/122392510-fbad3f80-cf84-11eb-8519-1f2f15397a91.png)

**Comment by spnzig** [comment]
I was initially facing a similar issue. Which version of PySyft are you using? Could you share in some environment details too.

**Comment by rkarn** [comment]
I am using pysft version 0.3.0. I am running it on ubuntu machine python version 3.6.9. 
>>> import syft
>>> syft.__version__
'0.3.0'

**Comment by tudorcebere** [comment]
Can you update to the latest version of Syft?

**Comment by rkarn** [comment]
root@ip-172-31-26-203:~# pip3 install syft==0.5.0rc2
ERROR: Could not find a version that satisfies the requirement syft==0.5.0rc2 (from versions: 0.1.0a1, 0.1.1a2, 0.1.2a1, 0.1.3a1, 0.1.4a1, 0.1.4a2, 0.1.5a1, 0.1.6a1, 0.1.7a1, 0.1.8a1, 0.1.9a1, 0.1.10a1, 0.1.10a2, 0.1.10a4, 0.1.11a1, 0.1.12a1, 0.1.13a1, 0.1.14a1, 0.1.15a1, 0.1.16a1, 0.1.19a1, 0.1.20a1, 0.1.21a1, 0.1.22a1, 0.1.23a1, 0.1.24a1, 0.1.25a1, 0.1.26a1, 0.1.27a1, 0.1.28a1, 0.1.29a1, 0.2.0a1, 0.2.0a2, 0.2.1a1, 0.2.2a1, 0.2.3a1, 0.2.3a2, 0.2.3a3, 0.2.3, 0.2.4, 0.2.5, 0.2.6, 0.2.7, 0.2.8, 0.2.9, 0.3.0, 0.5.0rc1)
ERROR: No matching distribution found for syft==0.5.0rc2


I tried with version "0.5.0rc1" but it has thrown another error as below:
ERROR: syft-proto 0.5.3 has requirement protobuf>=3.12.2, but you'll have protobuf 3.12.0 which is incompatible.

Then I tried updating probut. It results in further error:
root@ip-172-31-26-203:~# pip3 install protobuf==3.12.2
Collecting protobuf==3.12.2
Requirement already satisfied: setuptools in /usr/local/lib/python3.6/dist-packages (from protobuf==3.12.2) (51.3.3)
Requirement already satisfied: six>=1.9 in /usr/local/lib/python3.6/dist-packages (from protobuf==3.12.2) (1.15.0)
ERROR: tensorflow 1.15.5 has requirement numpy<1.19.0,>=1.16.0, but you'll have numpy 1.19.5 which is incompatible.
ERROR: coremltools 2.0 has requirement six==1.10.0, but you'll have six 1.15.0 which is incompatible.
ERROR: autokeras 1.0.11 has requirement tensorflow>=2.3.0, but you'll have tensorflow 1.15.5 which is incompatible.
Installing collected packages: protobuf
  Attempting uninstall: protobuf
    Found existing installation: protobuf 3.12.0
    Uninstalling protobuf-3.12.0:
      Successfully uninstalled protobuf-3.12.0
Successfully installed protobuf-3.12.2
